llvm/lib/Transforms/IPO/WholeProgramDevirt.cpp

```diff
@@ -1571,6 +1597,11 @@ void DevirtModule::applyICallBranchFunnel(VTableSlotInfo &SlotInfo,
   Apply(SlotInfo.CSInfo);
   for (auto &P : SlotInfo.ConstCSInfo)
     Apply(P.second);
+  for (auto &[F, C] : FunctionEntryCounts) {
+    assert(!F->getEntryCount(/*AllowSynthetic=*/true) &&
```

Comment:
It is false because this is a func that was initially set to have unknown earlier?

Reply:
it's set to `unknown` after this (==`applyICallBranchFunnel`) is called, if we didn't have module-internal callsites to use.

but to the original question, the functions appearing here are funnels we just generated, so should have no entry count.